README.md.template

```diff
@@ -53,17 +58,37 @@ nvm use lts/hydrogen
 
 * Run `npm install -g @hashgraph/solo`
 
-## Setup Kubernetes cluster
+## Quick Start Guide
+User can use one of the following three command to quickly deploy a standalone test network.
```

Comment:
Nathan I were talking about this.  This is more of a quick start for a developer who has checked out our repo and has all of the necessary pre-reqs.

The readme page goes on our npmjs page, and would be misleading, since I don't believe the task command would work out of the box without first installing Task somehow.  and then would require everything to be built, etc.

Reply:
Add instructions to install dependencies before using task